src/services/mongoose/accounts.ts

```diff
@@ -34,8 +34,8 @@ export const AccountsRepository = (): IAccountsRepository => {
     walletId: WalletId,
   ): Promise<Account | RepositoryError> => {
     try {
-      const result: UserType = await User.findOne({ _id: walletId }, projection)
-      if (!result) return new CouldNotFindError()
+      const result: UserType = await User.findOne({ walletId }, projection)
+      if (!result) return new CouldNotFindError("Invalid wallet")
```

Comment:
We don't usually parse the arguments from these sorts of errors, so if we'd like to add additional context for the final error-map we'd usually make a new distinct error type and add it to the error-map switch.